test/integration/containerslots-filter.bats

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function teardown() {
+	swarm_manage_cleanup
+	stop_docker
+}
+
+@test "containerslots filter" {
+    export DOCKER_DAEMON_ADDITIONAL_PARAMETERS="--label containerslots=2"
+	start_docker_with_busybox 2
```

Comment:
you can use `start_docker_with_busybox 2 --label containerslots=2` no need for `DOCKER_DAEMON_ADDITIONAL_PARAMETERS`